src/main.c

```diff
@@ -662,8 +662,9 @@ void HEADER_PRINT()
 
 int main(int argc, const char** argv)
 {
-    int img_idx = atoi(argv[1]);
-    const char *img_path = argv[2];
+    int img_idx = 0;
+    const char *img_path = NULL;
+    uint8_t *img_buff = NULL;
```

Comment:
log_print(LOG_ERR, "Failed to create %s\n: ", plock_file_path, strerror(errno));
Lost an %s in log_print format

Reply:
Done